### ISSUE TITLE
Local component does'nt shows in suggestions

### ISSUE THREAD

**Issue by mohammad0-0ahmad** [microsoft/vscode#133267]
Issue Type: <b>Bug</b>

Project type : React.

I am trying to import a local component that I've created which have the same name of a component of a dependency.
The local component does'nt shows in suggestions, only dependency component shows as possible option in suggustions list.

I've reinstalled the previous version "1.59"  and now it's working as fine as it should.

VS Code version: Code 1.60.1 (83bd43bc519d15e50c4272c6cf5c1479df196a4d, 2021-09-10T17:07:10.714Z)
OS version: Windows_NT x64 10.0.19041
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 3700X 8-Core Processor              (16 x 3600)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.92GB (19.31GB free)|
|Process Argv|--crash-reporter-id 4c393906-205f-4dae-b03a-84eba8fd18cb|
|Screen Reader|no|
|VM|40%|
</details><details><summary>Extensions (23)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-browser-preview|auc|0.7.1
http-status-codes|bea|1.0.2
vscode-eslint|dba|2.1.25
githistory|don|0.6.18
es7-react-js-snippets|dsz|3.1.1
prettier-vscode|esb|8.1.0
comment-anchors|Exo|1.9.6
vscode-graphql|Gra|0.3.18
vscode-test-explorer|hbe|2.21.1
vscode-env|Iro|0.1.0
vscode-lottie|lot|1.0.3
markdown-shortcuts|mdi|0.12.0
test-adapter-converter|ms-|0.1.2
color-highlight|nau|2.5.0
indent-rainbow|ode|8.0.0
material-icon-theme|PKi|4.10.0
calc|rai|2.0.1
vscode-xml|red|0.18.0
markdown-preview-enhanced|shd|0.6.0
comment-divider|sta|0.4.0
code-spell-checker|str|2.0.5
code-spell-checker-swedish|str|0.1.9
vscode-todo-highlight|way|1.0.4

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256860
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551:30345470
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscoreces:30358480
pythondataviewer:30285071
pythonvsuse255:30340121
vscod805cf:30301675
pythonvspyt200:30340761
vscextlangct:30333562
binariesv615:30325510
vsccppwtct:30364498
pythonvssor306:30344512
bridge0708:30335490
pygetstartedt2:30360495
bridge0723:30353136
pythonrunftest32cf:30365367
pythonf5test824cf:30361778
javagetstartedt:30364666
pythonvspyt187cf:30365362
pydsgst2:30361792
vscus224:30367819
vssid140cf:30363604
vssur157:30367808

```

</details>

**Comment by mjbvz** [comment]
Please share a minimal example project along with steps to reproduce the issue

**Comment by mohammad0-0ahmad** [comment]
Example project link :
 <https://github.com/mohammad0-0ahmad/vsc-bug>

### Current behavior in version 1.60

<img align="center" src="https://github.com/mohammad0-0ahmad/vsc-bug/blob/master/v_1.60.PNG?raw=true"/>
<br/>
<br/>

### Expected behavior in version 1.59

<img align="center" src="https://github.com/mohammad0-0ahmad/vsc-bug/blob/master/v_1.59.PNG?raw=true"/>

**Comment by mjbvz** [comment]
Thanks for the example project. However I can't reproduce this in my testing

Does this happen with all extensions disabled? Specifically make sure the lit-plugin extension is disabled

**Comment by mohammad0-0ahmad** [comment]
Yes, this happens with all extensions disabled and I've faced same issue on another pc.

**Comment by mjbvz** [comment]
Cam you please collect the TS Server log using your example repo:

1. Set `"typescript.tsserver.log": "verbose",`
1. Restart VS Code and reproduce the problem
1. In VS Code, run the `TypeScript: Open TS Server log` command
1. This should open a large log file called `tsserver.log`

**Comment by mohammad0-0ahmad** [comment]
Sorry for being late with answer but I had to install the new version again.
I've disabled all extensions before doing that you mentioned in the latest post.
tsserver.log file link : 
<https://github.com/mohammad0-0ahmad/vsc-bug/blob/master/tsserver.log>

**Comment by mohammad0-0ahmad** [comment]
I've created new branch without MUI.
Then I've created  the following components:
src/Typography2.tsx
src/example/Typography.tsx
src/example/Typography3.tsx

I've got the old Typography as suggestion but not the new Typography component "src/example/Typography.tsx".

### Current behavior V1.60
<img align="center" src="https://github.com/mohammad0-0ahmad/vsc-bug/blob/without-MUI/v_1.60.PNG?raw=true"/>

### tsserver.log file link :
https://github.com/mohammad0-0ahmad/vsc-bug/blob/without-MUI/tsserver.log

### Expected behavior  V1.59
<img align="center" src="https://github.com/mohammad0-0ahmad/vsc-bug/blob/without-MUI/v_1.59.PNG?raw=true"/>

Branch link:
https://github.com/mohammad0-0ahmad/vsc-bug/tree/without-MUI

I hope it will be helpful.

**Comment by mohammad0-0ahmad** [comment]
Sorry for being late with answer again. 
Yes it's work now after installing the new vsc 1.62.1 version and the mentioned extension in your comment.
